src/vs/workbench/api/browser/mainThreadTunnelService.ts

```diff
@@ -61,6 +61,9 @@ export class MainThreadTunnelService extends Disposable implements MainThreadTun
 				const forward = this._proxy.$forwardPort(tunnelOptions, tunnelCreationOptions);
 				if (forward) {
 					return forward.then(tunnel => {
+						if (!tunnel) {
+							return undefined;
+						}
 						return {
```

Comment:
```suggestion
						return !tunnel ? undefined :  {
```

Can simplify it ...!